pkg/urbit/vere/io/term.c

```diff
@@ -143,60 +148,33 @@ u3_term_log_init(void)
       uv_pipe_open(&(uty_u->pop_u), uty_u->fid_i);
     }
 
-    //  Load terminfo strings.
+    //  configure output escape sequences
+    //
+    //    (as reported by the terminfo database we bundled)
```

Comment:
I don't understand this comment. The sequences below are literals in our code, not loaded in from terminfo, right? Where does the terminfo database file come into play?

Reply:
Originally, the urbit binary was built on/far specific environments and we used the local terminfo database. When we switched to static binaries, there were cross-compilation and compatibility issues, so we bundled a terminfo database. But our escape code usage is very minimal and broadly compatible, so I removed the terminfo integration entirely in v0.10.8. I'll update the comments.